nephio/optional/nephio-stock-repos/repo-oai-core-packages.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: Repository
+metadata:
+  name: oai-core-packages
+  namespace: default
+  labels:
+    kpt.dev/repository-access: read-only
+    kpt.dev/repository-content: external-blueprints
+spec:
+  content: Package
+  deployment: false
+  git:
+    branch: r2
```

Comment:
should this be main? Should these move to `catalog/workloads/oai`?

Reply:
@johnbelamaric I don't think so there is any need to move them in catalog, then we have to fork from OAI repository. I kept `r2` just for not later I will merge to `main` and tag it before we will do cut off.